### PR TITLE
Allow baksnapper to run without dbus

### PR DESCRIPTION
**Pull request by Gooberpatrol66**

### CODE REVIEW

**Comment by plattfot** [comment]
Thank you for taking the time to create pull request.

The documentation for the `--no-dbus, mention this`:

```
Use with caution since a running snapperd will not know about modifications made to the system.
```
See [manpage](http://snapper.io/manpages/snapper.html). Will this option create an edge case where it can return the wrong subvolume?

Also what's the use case running this without dbus?

**Comment by Gooberpatrol66** [comment]
`Use with caution since a running snapperd will not know about modifications made to the system.`

I'm not sure what that means, specifically. What modifications?

`Also what's the use case running this without dbus?`

Probably useful for disaster recovery (http://snapper.io/2016/12/19/no-dbus-for-all.html)
Personally, I packaged snapper for a distro, and it's not working with dbus, and just removing the dependency seems easier than debugging it, as it doesn't use that functionality.

**Comment by plattfot** [comment]
No idea what modifications they're talking about either.

I doubt fetching the subvolume from the config is impacted. If things start breaking this should be easy to rollback.

And I'm planning on removing the snapper dependency in the future (and a rewrite to get better testing).

Can you add 2022 to your name in the copyright header and I'll merge this?

File: baksnapper.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@
 # btrfs' incremental send and receive
 
 # Copyright (C) 2015-2021  Fredrik Salomonsson <plattfot@posteo.net>
-# Copyright (C) 2021  Nathan Dehnel
+# Copyright (C) 2021-2022  Nathan Dehnel
```
Comment (plattfot) [inline baksnapper.sh:7]: Please revert this change, as the PR only affects the `baksnapperd.sh` file.
